website/client/components/chat/copyAsTodoModal.vue

```diff
@@ -36,9 +36,18 @@ export default {
       task: {},
     };
   },
+  generateURL () {
```

Comment:
you should add `message` to `generateURL` or else you won't have a message to use ^^